velox/exec/tests/SharedArbitratorTest.cpp

```diff
@@ -3859,14 +3859,20 @@ TEST_F(SharedArbitrationTest, concurrentArbitration) {
           } else if ((i % 4) == 1) {
             task =
                 runOrderByTask(vectors, newQueryCtx(queryCapacity), numDrivers);
+          } else if ((i % 4) == 2) {
+            task = runRowNumberTask(
+                vectors, newQueryCtx(queryCapacity), numDrivers);
           } else {
             task = runTopNTask(vectors, newQueryCtx(queryCapacity), numDrivers);
           }
         } catch (const VeloxException& e) {
-          VELOX_CHECK(
+          bool expected =
```

Comment:
Looks like this code has the same problem of not ensuring that the code in 'try' actually throws.